weblate/addons/forms.py

```diff
@@ -652,3 +662,32 @@ class PropertiesSortAddonForm(BaseAddonForm):
         required=False,
         initial=False,
     )
+
+
+class ChangeBaseAddonForm(BaseAddonForm):
+    """Base form for Change-based addons."""
+
+    events = forms.MultipleChoiceField(
+        label=gettext_lazy("Change events"),
+        required=False,
+        widget=SortedCheckboxSelectMultiple(),
```

Comment:
Maybe use `SortedSelectMultiple` instead? It might be a better UX to select the events.